staging/src/k8s.io/apiextensions-apiserver/hack/update-codegen.sh

```diff
@@ -21,13 +21,20 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
+
 # generate the code with:
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 CLIENTSET_NAME_VERSIONED=clientset \
-CLIENTSET_NAME_INTERNAL=internalclientset \
-${CODEGEN_PKG}/generate-internal-groups.sh all \
+${CODEGEN_PKG}/generate-groups.sh deepcopy,client,lister,informer \
```

Comment:
I think you don't need `deepcopy` here, because `generate-internal-groups.sh` takes care of it?

Reply:
make sense